### PR TITLE
change a reference link in docs

### PR DESCRIPTION
**Pull request by Detachment**
original link is invalid, add 'docs/' path can make it done~

### CODE REVIEW

File: docs/spec/direct.zh-CN.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ title: 直截了当
 
 多字段行内编辑
 
->注：在『多字段行内编辑』的情况下，显示的内容和编辑时所需的字段会存在比较大的差异，所以更需要『巧用过渡』原则中的[『解释刚刚发生了什么』](../spec/transition#解释刚刚发生了什么)来消除这种视觉影响。
+>注：在『多字段行内编辑』的情况下，显示的内容和编辑时所需的字段会存在比较大的差异，所以更需要『巧用过渡』原则中的[『解释刚刚发生了什么』](../docs/spec/transition#解释刚刚发生了什么)来消除这种视觉影响。
```
Comment (paranoidjk) [inline docs/spec/direct.zh-CN.md]: @Detachment can you fix en-US.md at the same time?

**Review by paranoidjk — CHANGES_REQUESTED** [review]

**Comment by paranoidjk** [comment]
@Detachment you don't need to close, just continue push your code to this branch.

**Comment by Detachment** [comment]
@paranoidjk done~

File: docs/spec/direct.zh-CN.md (hunk 1 above)
Comment (Detachment) [inline docs/spec/direct.zh-CN.md]: done~

**Pull request merged**